### PR TITLE
Add jQuery-Autocomplete registry and override

### PR DESCRIPTION
**Pull request by smajl**

### CODE REVIEW

File: package-overrides/github/devbridge/jQuery-Autocomplete@1.3.0.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  "files": [
+    "dist/",
+    "typings/jquery-autocomplete/",
+    "readme.md"
+  ],
+  "dependencies": {
+    "jquery": "*"
+  },
+  "shim": {
+    "jquery.autocomplete": ["jquery"]
```
Comment (aluanhaddad) [inline package-overrides/github/devbridge/jQuery-Autocomplete@1.3.0.json]: I have tested this with both the latest stable and beta releases of JSPM and the `"shim"` section is not necessary as this is a well behaved package that only binds to a global `jQuery` as a last resort. Simply having jquery as a dependency cause the right behavior in both 2.2.4 and 3.1.1 releases

File: package-overrides/github/devbridge/jQuery-Autocomplete@1.3.0.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+  "files": [
+    "dist/",
+    "typings/jquery-autocomplete/",
```
Comment (aluanhaddad) [inline package-overrides/github/devbridge/jQuery-Autocomplete@1.3.0.json]: Awesome that you included the typings, that is much appreciated. However, there is an issue. The typings directory includes a file named `jquery.autocomplete-tests.ts` which writes directly into the global scope.

File: package-overrides/github/devbridge/jQuery-Autocomplete@1.3.0.json (hunk 1 above)
Comment (smajl) [inline package-overrides/github/devbridge/jQuery-Autocomplete@1.3.0.json]: Thanks for the test, I have removed the shim.

File: package-overrides/github/devbridge/jQuery-Autocomplete@1.3.0.json (hunk 2 above)
Comment (smajl) [inline package-overrides/github/devbridge/jQuery-Autocomplete@1.3.0.json]: Changed, thanks for finding this out.

**Comment by smajl** [comment]
Requested changes made. :)

**Comment by aluanhaddad** [comment]
Thanks @smajl!

**Pull request merged**